code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -147,12 +147,14 @@ export const ProgressDot = React.createClass({
 
   render() {
     const level = this.props.level;
-    const uid = level.uid || level.id.toString();
+    const onCurrent = this.props.currentLevelId &&
```

Comment:
when I see this much code I look for a comment explaining briefly what it does :)